lib/services/firebase_writer_service.dart

```diff
@@ -34,4 +34,25 @@ class FirebaseWriterService {
       return left("Error");
     }
   }
+
```

Comment:
Why in line 16 is either string or string?

Reply:
it looks like it could be just a string